apps/worker/src/monitor/status.monitor.module.ts

```diff
@@ -54,24 +54,12 @@ import { QueueConstants } from '../../../../libs/common/src';
       {
         name: QueueConstants.TRANSACTION_RECEIPT_QUEUE_NAME,
         defaultJobOptions: {
-          attempts: 3,
```

Comment:
this was not working and had to add it to the job itself

Reply:
Good to know, we should throw a ticket about enhancement of queue configs

Reply:
I think we have a ticket for it